gcp-repo/metadata.p2.inf

```diff
@@ -39,6 +39,8 @@ units.1.properties.4.name=org.eclipse.equinox.p2.contact
 units.1.properties.4.value=https://github.com/GoogleCloudPlatform/google-cloud-eclipse
 units.1.properties.5.name=org.eclipse.equinox.p2.doc.url
 units.1.properties.5.value=https://cloud.google.com/eclipse/docs/
+units.1.properties.5.name=org.eclipse.equinox.p2.description
+units.1.properties.5.value=Support development of Google Cloud Platform projects in Eclipse
```

Comment:
`properties.6`?

`Development support`? Otherwise, I feel like it should be `Supports development`.

Reply:
I stared at these lines and wondered why I needed .6 on the YEdit and Docker-Editor lines but not here. :blush:

Will go for _Supports development_ for now.